### PR TITLE
French translation of the CSS Counters Styles API reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates the French translation of the CSS Counters Styles API reference.

### CODE REVIEW

File: files/fr/web/api/css_counter_styles/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Module CSS Styles de compteurs
```
Comment (SphinxKnight) [inline files/fr/web/api/css_counter_styles/index.html:2]: I'm torn between the English "canonical term" for the name of the spec and the French equivalent here.
Maybe adding '(Counter Styles)' (within parenthesis) afterwards is a good compromise. I wonder how (French-speaking) people will discover this page: will they search for "counter style css" in their search engine, etc.

Let me know if you disagree or think of a better solution.

File: files/fr/web/api/css_counter_styles/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+slug: Web/API/CSS_Counter_Styles
+translation_of: 'Web/API/CSS_Counter_Styles'
+---
+<div>{{DefaultAPISidebar("CSS Counter Styles")}}{{SeeCompatTable}}</div>
+
+<p>Le module CSS Styles de compteurs permet de définir des styles de compteurs personnalisés qui peuvent être utilisés pour les marqueurs de listes CSS et les compteurs des contenus générés.</p>
```
Comment (SphinxKnight) [inline files/fr/web/api/css_counter_styles/index.html:8]: As this is inline, I would definitely add the English version in parenthesis here.

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
A slight """""debate"""" about adding the English name of the spec but apart from that: good to go.

File: files/fr/web/api/css_counter_styles/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/api/css_counter_styles/index.html]: Agreed. New proposal in https://github.com/mdn/translated-content/pull/1766/commits/b819ee39bcf12645dec134db59438b571dbf2f48

File: files/fr/web/api/css_counter_styles/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/api/css_counter_styles/index.html]: Agreed. Let's add both, then. New proposal in https://github.com/mdn/translated-content/pull/1766/commits/b819ee39bcf12645dec134db59438b571dbf2f48

**Review by SphinxKnight — APPROVED** [review]
Thanks :)

**Pull request merged**